### ISSUE TITLE
Spinner - Keyboard Trap

### ISSUE THREAD

**Issue by weiprd** [primefaces/primefaces#7815]
In order to be accessible to all users, a website or web application has to fulfill a list of WCAG 2.1 success criteria. The Spinner in the Diamond theme does not fulfill success criterion [2.1.2](https://www.w3.org/TR/WCAG21/#no-keyboard-trap).
If you move through the fields of the Input-Showcase using the tab key, you are not able to leave the field of the spinner.  

**Environment** 
Diamond Theme
Problem occures on Chrome, Firefox and Edge.

**To Reproduce**

1. Go to the input showcase of the Diamond theme https://www.primefaces.org/diamond/input.xhtml
2. Navigate through the site by using the tab key,
3. If the focus is in the field of the spinner, you are not able to leave it by using tab or shift + tab

**Expected behavior**
It should be possible to navigate forward by tab and backward by shift + tab.

**Comment by melloware** [comment]
Its not diamond theme related looks like it happens on the showcase as well: https://www.primefaces.org/showcase/ui/input/spinner.xhtml

**Comment by melloware** [comment]
duplicate of https://github.com/primefaces/primefaces/issues/7378 see that ticket for MonkeyPatch workaround.
